osu.Game/Rulesets/Mods/ModUsage.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Mods
+{
+    /// <summary>
+    /// The usage of this mod to determine whether it's playable in such context.
+    /// </summary>
+    public enum ModUsage
+    {
+        /// <summary>
+        /// Used for a per-user gameplay session.
+        /// Determines whether the mod is playable by an end user.
+        /// </summary>
+        User,
+
+        /// <summary>
+        /// Used in multiplayer but must be applied to all users.
```

Comment:
I suggest renaming these xmldocs to read more like: `The mod can be used in multiplayer but ...`